pi/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Original source: https://gist.github.com/pepyakin/2ff227c2d837a2eacd8d3879d5e0c94f
+# Includes some customizations for polymesh
+
+FROM rust:latest
+
+RUN dpkg --add-architecture armhf && \
+    apt-get update && apt-get upgrade -y && \
+    apt-get install -y aptitude && \
+    aptitude install -y \
+        gcc-arm-linux-gnueabihf \
+        g++-arm-linux-gnueabihf \
+        pkg-config \
+        cmake \
+        libssl-dev \
+        git \
+        clang \
+        libclang-dev \
+        libssl-dev:armhf
+
+# Install nightly with w32-u-u and add arvm7 to stable
+RUN rustup target add armv7-unknown-linux-gnueabihf
```

Comment:
If you are targeting Pi 3, wouldn't it be better to compile for ARMv8, target `aarch64-unknown-linux-gnu`?

Reply:
Pi3 runs the armv7 instruction set by default. It requires a "hack" to switch to armv8 instruction set. Pi2 can not use armv8 at all afaik. Since there isn't much performance improvement in armv8 instruction set over armv7 I don't think it's worth the hassle.

Reply:
OK but it would be nice to compare in the future. ARMv7 is 32-bit while ARMv8 is 64-bit and supports wider scalar operations.

Reply:
Probably a hack is not required any more since Raspbian supports 64-bit: https://medium.com/@kavoshex/raspian-added-support-for-64-bit-kernel-6cf27c2b26db

Reply:
Feel free to test it when you get your pi :).

I am just happy it works at all xD. Probably not gonna run it for long anyway.